tidy3d/components/data/data_array.py

```diff
@@ -71,16 +71,23 @@ class DataArray(xr.DataArray):
     _data_attrs: Dict[str, str] = {}
 
     def __init__(self, data, *args, **kwargs):
-        """Initialize ``DataArray``."""
-
-        # initialize with untraced data
-        super().__init__(getval(data), *args, **kwargs)
-        # and put tracers in .attrs
+        super().__init__(data, *args, **kwargs)
         if isbox(data):
             self.attrs[AUTOGRAD_KEY] = data
 
     @property
-    def tracers(self) -> Box:
+    def has_tracers(self) -> bool:
+        return self.tracers is not None
+
+    @property
+    def static(self):
+        vals = self.values if self.values.size == 0 else np.vectorize(getval)(self.values)
```

Comment:
does `getval` need a wrapper in `utils.py`? I'm not 100% sure about what happens here if `self.values` is a `np.ndarray` of `dtype=object`. my impression is that getval will not work here unless we apply it to `np.ndarray(self.values.tolist())` but could be wrong.

Reply:
ah, maybe the vectorize already takes care of this.

Reply:
yeah the vectorize is there exactly for that purpose. `np.array(self.values.tolist())` doesn't work actually

Reply:
ah, good to know. Does this pop up anywhere else? (ie where the `anp.array(...tolist())` was called previously?) if so, maybe we should refactor it into one helper function

Reply:
sorry, got a bit confused.. never mind